src/apps/chifra/pkg/base/address.go

```diff
@@ -18,8 +18,7 @@ func (a *Address) Hex() string {
 	if a.IsZero() {
 		return "0x0"
 	}
-	// This is 1000 ns/op faster than strings.ToLower
-	return "0x" + hex.EncodeToString(a.Bytes())
+	return bytesToAddressString(a.Address.Bytes())
```

Comment:
We have an issue for this and as long as it's all centralized inside of Hex() we can test this more carefully later.